service/history/execution/mutable_state_builder.go

```diff
@@ -357,10 +358,23 @@ func (e *mutableStateBuilder) Load(
 				// feature is tested and/or we have mechanisms in place to deal
 				// with these types of errors
 				e.metricsClient.IncCounter(metrics.WorkflowContextScope, metrics.MutableStateChecksumMismatch)
-				e.logError("mutable state checksum mismatch", tag.Error(err))
+				e.logError("mutable state checksum mismatch",
```

Comment:
I know this file doesn't have good enough test coverage already but these new lines will drop new line coverage metric significantly. Can you add some basic tests to cover these lines? we can expand on the coverage later on